attendance/views.py

```diff
@@ -1,11 +1,26 @@
-from django.http import HttpResponse, JsonResponse
+from django.http import HttpResponse
+from django.http import JsonResponse
+from django.http import HttpResponseRedirect
```

Comment:
could we agree that listing each import, although a little annoying, is beneficial to check where each import comes from, to help flake8 know if we are importing unused stuff, in diffs it would show only the import you're adding, etc. ?